sqlmesh/core/model/definition.py

```diff
@@ -2464,6 +2464,24 @@ def _create_model(
 
     statements: t.List[t.Union[exp.Expression, t.Tuple[exp.Expression, bool]]] = []
 
+    # Merge default pre_statements with model-specific pre_statements
+    if "pre_statements" in defaults:
+        kwargs["pre_statements"] = [
+            exp.maybe_parse(stmt, dialect=dialect) for stmt in defaults["pre_statements"]
+        ] + kwargs.get("pre_statements", [])
```

Comment:
Provided the value can be `None` shouldn't this be `kwargs.get(...) or []`? Or otherwise, shouldn't the default value be `[]` in the config?